be/src/exec/schema_scanner/schema_fe_tablet_schedules_scanner.cpp

```diff
@@ -0,0 +1,177 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "exec/schema_scanner/schema_fe_tablet_schedules_scanner.h"
+
+#include "exec/schema_scanner/schema_helper.h"
+#include "gutil/strings/substitute.h"
+#include "runtime/string_value.h"
+#include "storage/tablet.h"
+#include "types/logical_type.h"
+
+namespace starrocks {
+
+SchemaScanner::ColumnDesc SchemaFeTabletSchedulesScanner::_s_columns[] = {
+        {"TABLE_ID", TYPE_BIGINT, sizeof(int64_t), false},
+        {"PARTITION_ID", TYPE_BIGINT, sizeof(int64_t), false},
+        {"TABLET_ID", TYPE_BIGINT, sizeof(int64_t), false},
+        {"TYPE", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"PRIORITY", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"STATE", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"STATUS", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"CREATE_TIME_MS", TYPE_BIGINT, sizeof(int64_t), false},
+        {"SCHEDULE_TIME_MS", TYPE_BIGINT, sizeof(int64_t), false},
+        {"FINISH_TIME_MS", TYPE_BIGINT, sizeof(int64_t), false},
+        {"CLONE_SRC_BE", TYPE_BIGINT, sizeof(int64_t), false},
+        {"CLONE_DEST_BE", TYPE_BIGINT, sizeof(int64_t), false},
+        {"CLONE_BYTES", TYPE_BIGINT, sizeof(int64_t), false},
+        {"CLONE_TIME_MS", TYPE_BIGINT, sizeof(int64_t), false},
```

Comment:
`CLONE_TIME_MS` -> `CLONE_TIME` The granularity is second.

Reply:
rename to clone_time and change to double